### ISSUE TITLE
Fix react-select label association on trainers page

### ISSUE THREAD

**Issue by dahliacreative** [fitnassist/fitnassist#4]
The training location Select on /trainers still fails the axe 'label' rule because react-select's dummy input doesn't pick up the inputId when isSearchable is changed. Needs a proper fix.

**Comment by dahliacreative** [comment]
Fixed — all Select components now have proper htmlFor/inputId label associations.
